src/components/Navigation.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react"
+import { Link } from "gatsby"
+import "../sass/Navigation.scss"
```

Comment:
⭐ _(nice to have)_ js generally starts filename with a lowercase letter

```suggestion
import "../sass/navigation.scss"
```

Reply:
Changed all file names to lowercase, both js and scss  ✅